data/hai/unfettered 1 war preparations.txt

```diff
@@ -0,0 +1,1550 @@
+			choice
+				`	"I will depart as soon as I can."`
+					accept
+				`	"Is it common for the Unfettered to use heralds to bring messages?"`
+				`	"This seems too risky for me."`
+					decline
+			`	"We don't believe technical ways of communicating to be safe, and they break the link between us. Messages could easily be spied upon by anyone intercepting it, so for some confidential missions like this one, it is quite common to use heralds. On top of that, the warlords generally have better things to do than moving around giving missions all the time. Do you have enough information? What shall I say to my warlord?"`
+			choice
+				`	"I will depart as soon as I can."`
+					accept
+				`	"You could have just sent a message on a secure channel."`
+			`	"What is secure? There is no code that cannot be broken, and I did not have to travel far anyway. The Fettered Hai have become quite apt at languages, with their interest to communicate with other species like the Quarg or humans. Encryption is just like another language. We cannot risk them learning of this particular mission - you will understand why later. Information is the key here, and it always has been. We also don't want those north to learn from this. They would surely be able to decrypt it.`
+			`	We are not all as brutal as we let other races think. My warlord believes that it is to our advantage to not appear cunning, so that they may underestimate us." He seems to have had enough of talking, and is now waiting for you to answer He-Yi's request.`
+			choice
+				`	"I will depart as soon as I can."`
+					accept
+				`	"This seems too risky for me."`
+					decline
+	on waypoint
+		dialog `It seems that the Degenerate is in this system, but judging from the marks on its hull, it will likely land to make repairs and restock ammo soon. You may be able to get the item without too much risk that way, and bribing them will mean secure landing. Or you could always just board it now and be done with it. You could even sabotage the ship to make your job easier, or take some more valuables with you.`
+	npc board
+		personality staying target
+		to despawn
+			has "Unfettered: Special Cargo 1B: active"
+		government "Mhairi's Legion"
+		system "Alcyone"
+		ship "Leviathan (Cube)" "Degenerate"
+		dialog `You have managed to disable and board Mhairi's flagship. The item that fits best the description given is without any doubt the Mystery Cube. Bring it back to <destination>.`
+	npc
+		personality coward plunders vindictive staying
+		government "Mhairi's Legion"
+		system "Alcyone"
+		fleet "Large Northern Pirates" 3
+		fleet "Small Northern Pirates" 5
+	on visit
+		dialog `You don't have the item He-Yi asked for. Or you have not brought the right item back. If that's the case, go back and board the ship to take the right item.`
+	on complete
+		require "Mystery Cube"
+
+
+
+# you may refuse to set it up to not spread mistrust with the Hai.
+mission "Unfettered: Special Cargo 2"
```

Comment:
Is the mission where you actually find the cube in the previous file? Why?

Reply:
it's not in the previous file, there is part of it (stealing it from on board the landed ship) in the sides missions though caus you can just plunder it instead of landing

Reply:
I'm pretty sure it's in 0 (or some earlier file), not in side missions, as I have not reviewed that file yet.

Reply:
I'll double check but I think I saw the first part right above this

Reply:
What's the mission in mercenaries around line 678?

Reply:
It's the optional part of this, where you can get gold bars if you land (or just buy the cube)

Reply:
Maybe it would be useful if all parts of the mission chain would be in the same file, even if they are optional or only trigger on landing.

Reply:
mhhhh thing is I'm trying to split it up because thats a mercenary action